bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "uniq",
-  "version": "2.2.5",
+  "version": "2.3.0",
```

Comment:
Is this a breaking change? If so, 🔝, otherwise, bump last number instead of the middle one 👍